### DISCUSSION TITLE
Occupationalhazards event triggers with no NPC in sight

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14043] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Occupationalhazards event triggers by just walking into a room, and you talk to a non-existing NPC

The text then says the NPC sits into a chair

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/e188980b-8533-407a-bcda-0025a5d65764)
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/bc8c20a3-2575-431e-94c4-971631c14db5)

Use this tidbit of code from `miketheidiot` event

```xml
      <TagAction criteria="itemtag:chair" tag="chair" RequiredModuleTag="CrewModule" />
      <SpawnAction npcsetidentifier="outpostnpcs1" npcidentifier="miner" targettag="hardhatman" spawnlocation="Outpost" targetmoduletags="CrewModule" />
      <NPCOperateItemAction npctag="hardhatman" targettag="chair" Operate="true" />
      <TriggerAction target1tag="player" target2tag="hardhatman" applytotarget1="triggerer_player" waitforinteraction="true"/>
```

### Reproduction steps

generate mining outpost
triggerevent Occupationalhazards 
walk into crew module
converse with ghosts

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
dupe

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13912
